colorsys.py

```diff
@@ -52,7 +57,7 @@ def hls_to_rgb(hue, light, sat):
     )
 
 
-def _v(chroma1, chroma2, hue):
+def _v(chroma1: float, chroma2: float, hue: float) -> float:
```

Comment:
It's not really specifically related to this PR but I'm curious if you know what this `_v` function is doing?  I don't recognize the algorithm in it (tho I have little experience with light sensors and related things). 

I wonder if there is a more descriptive name that we could use for it.